meinberlin/react/contrib/ListCardPill.jsx

```diff
@@ -26,23 +21,20 @@ export const ListItemBadges = props => {
   }
 
   return (
-    <div className="list-item__labels">
+    <div className="list-card__status status">
```

Comment:
this section is actually where the status will also go as well i think according to the BerlinOnline styleguide but we can sort that out after the status PR is merged